### DISCUSSION TITLE
Trim values on FCTL EFIS Page

### DISCUSSION THREAD

**Discussion by OldGroof** [magknight/787aviatorsTracker#773] [Completed things] [answered]
From what I can find online the FCTL efis page displays the trim values all the time. And these values disappear at some point during the flight on the EICAS. See the attached picture for the efis page.

My guess would be the pitch trim values hide when the gear and/or flaps are up?

![image](https://user-images.githubusercontent.com/39658757/159107117-88b9c68d-fe04-4613-bb67-b1c5f2345c7c.png)

**Comment by aeroniemi** [comment, marked as answer]
Added in 1.7.42
